packages/react-components/src/localization/locales/en-US/strings.json

```
@@ -141,7 +141,8 @@
     "screenIsBeingSharedMessage": "You are sharing your screen",
     "screenShareLoadingMessage": "Loading {participant}'s screen",
     "localVideoLabel": "You",
-    "localVideoCameraSwitcherLabel": "Cycle camera",
-    "localVideoMovementLabel": "Movable Local Video Tile"
+    "localVideoCameraSwitcherLabel": "Switch camera",
+    "localVideoMovementLabel": "Movable Local Video Tile",
+    "localVideoSelectedDescription": "selected"
```

This string should have the `{cameraName}` inside it:
```suggestion
    "localVideoSelectedDescription": "{cameraName} selected"
```

It allows Contoso to understand this string is used in a string builder and how the string is built.
E.g. on L142 we have: `"screenShareLoadingMessage": "Loading {participant}'s screen",`

---

(TD must do some magic to make use of this, @mgamis-msft would have more info on that)

---

Love the magic! Changed it up to use {cameraName} and changed up ariaDescription to follow suit.

---

Thanks for making those changes!